app/code/Apruve/Payment/Controller/Data/Index.php

```diff
@@ -57,51 +58,52 @@ public function execute() {
             $discountItem['sku'] = self::DISCOUNT;
             $discountItem['variant_info'] = '';
             $discountItem['view_product_url'] = $this->helper->getStoreUrl();
-        
+
             $hasDiscountItem = false;
             foreach ($order['line_items'] as $k => $item) {
                 if ($item['title'] == self::DISCOUNT) {
                     $hasDiscountItem = true;
                 }
             }
-        
+
             if (!$hasDiscountItem) {
                 $order['line_items'][] = $discountItem;
             }
         } else {
             // Remove Discount item
             foreach ($order['line_items'] as $k => $item) {
                 if ($item['title'] == self::DISCOUNT) {
-                    unset ($order['line_items'][$k]);
+                    unset($order['line_items'][$k]);
                 }
             }
         }
 
         $this->order = $order;
-        $data = array(
+        $data = [
             'order' => json_encode($this->order),
```

Comment:
Is this going to be double-encoded now that the return value is also being encoded?